solr/core/src/java/org/apache/solr/cloud/Overseer.java

```diff
@@ -19,14 +19,7 @@
 import java.io.Closeable;
 import java.io.IOException;
 import java.lang.invoke.MethodHandles;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
please configure your editor to not do this

Reply:
I never run the `idea` task, IntelliJ seems to be able to import the gradle project natively for me. Occasionally I have to "restart and invalidate caches" to reimport it, but that's usually after I've been changing branches around a lot.

I couldn't find a checked in or shared IntelliJ settings file, although I feel like it used to exist?

Under the settings, `Editor > Code Style > Java` find the tab called `imports` and I have class count set to 99 and all of the `packages to use import with *` choices unchecked or deleted.